internal/pkg/manifest/svc.go

```diff
@@ -534,10 +534,10 @@ func (sidecar SidecarConfig) exposedPorts(sidecarName string) ([]ExposedPort, er
 	}
 	return []ExposedPort{
 		{
-			Port:          uint16(port),
-			Protocol:      strings.ToLower(protocol),
-			ContainerName: sidecarName,
-			Implicit:      true,
+			Port:                   uint16(port),
+			Protocol:               strings.ToLower(protocol),
+			ContainerName:          sidecarName,
+			IsContainerDefinedPort: true,
```

Comment:
😂 don't hate me but what about `isDefinedByContainer`, so that it reads:
```go
for _, portConfig := range exposedPorts.PortsForContainer[targetContainer] {
  if portConfig.isDefinedByContainer
```

also, it seems like this can be a private field, is that correct?